### PR TITLE
ICMSLST-2439

### PR DESCRIPTION
**Pull request by chris-pettinga**
Adding DBT logo watermark to certificate PDFs

### CODE REVIEW

**Comment by mikeylockwood** [comment]
The positioning and colour of the watermark image is a little off.

V1

image


V2
image

I think the document size and header styling needs to be corrected to see how the watermark appears on the page.

File: web/templates/pdf/shared/certificate-base.html [hunk 1]
```diff
@@ -3,76 +3,81 @@
...
+			{{ get_css_rules_as_string('web/css/pdfs/licence-and-certificate-common-style.css')|safe }}
+			{{ get_css_rules_as_string('web/css/pdfs/certificate-style.css')|safe }}
+			{{ get_css_rules_as_string('web/css/components/signature.css')|safe }}
+			.dbt_logo_watermark {
+          background-image: url("data:image/png;base64,{{ get_file_base64('web/img/dbt_logo.png') }}");
+      }
+		</style>
+		{% block stylesheet %}{% endblock %}
+		<title>{% block page_title %}{{ page_title }}{% endblock %} - ICMS</title>
+	</head>
```
Comment (mikeylockwood) [inline web/templates/pdf/shared/certificate-base.html:17]: Tabs to 2 spaces please

**Comment by chris-pettinga** [comment]
> The positioning and colour of the watermark image is a little off.
> 
> V1
> 
> image
> V2 image
> 
> I think the document size and header styling needs to be corrected to see how the watermark appears on the page.

Thanks for the review @mikeylockwood - I think the different colour is because V1 used the low-resolution logo and blew it up to create the watermark which resulted in a lighter grey and looser edges. Basically I'm trying to argue that although it looks different, V2 looks better. However I know that kind of goes against the die-hard lift-and-shift philosophy so I'm open to change.

**Comment by chris-pettinga** [comment]
@mikeylockwood I have shrunken the header logo and pushed the watermark image a little bit further down so it matches V1 more closely.

File: web/tests/domains/case/_import/test_views.py [hunk 2]
```diff
@@ -111,7 +111,7 @@ def test_preview_cfs_certificate(ilb_admin_client, cfs_app_submitted):
     pdf = response.content
     assert pdf.startswith(b"%PDF-")
     # ensure the pdf generated has some content
-    assert 5000 < len(pdf) < 150000
+    assert 5000 < len(pdf) < 200000
```
Comment (MattHolmes123) [inline web/tests/domains/case/_import/test_views.py:114]: Ive always loved these tests... assert "they contain something" :)

**Review by MattHolmes123 — APPROVED** [review]

**Review by mikeylockwood — APPROVED** [review]

**Pull request merged**